readthedocs/projects/tasks.py

```diff
@@ -1440,6 +1454,8 @@ def webhook_notification(version, build, hook_url):
         'slug': project.slug,
         'build': {
             'id': build.id,
+            'commit': build.commit,
+            'state': build.state,
```

Comment:
We should define what data we send here in the docs as part of this PR, it will be much more useful to users that way :)